package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/redbadger/immutable-cursor",
   "dependencies": {
-    "atom-store": "0.0.2",
-    "immutable": "3.7.6"
+    "atom-store": "0.0.3",
+    "immutable": "^3.7.6"
```

Comment:
Nice catch. I missed that in https://github.com/redbadger/immutable-cursor/pull/13/commits/3b40e2e78e9e022e6829ba89671ef65c45639c59